src/redux/categories/categoriesSlice.js

```diff
@@ -10,7 +11,7 @@ const categoriesSlice = createSlice({
   reducers: {
     checkStatus: (state) => {
       const newState = { ...state };
-      newState.categories = [...newState.categories, 'Under construction'];
+      newState.categories = [...newState.categories, 'Status checked'];
```

Comment:
- [ ] When clicking the button on categories pages the message is not being shown, kindly check that